src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -4,49 +4,93 @@
  * <p>Реалізувати свій ArrayList який імплементує інтерфейс List</p>
  */
 public class ArrayList<T> implements List<T> {
+    private static final int init_capacity = 10;
```

Comment:
```suggestion
    private static final int INIT_CAPACITY = 10;
```

Reply:
yes, at first i also wanted to name constants like that, but StyleChecker didn't allow me this

Reply:
@SkaterWeather I have cloned your solution, renamed to INIT_CAPACITY: run the mvn package and everything works well. So comment from @Sofasmile is suitable